source/Nevermore.IntegrationTests/RelationalTransaction/TransactionTimerFixture.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using Nevermore.Advanced;
+using Nevermore.Diagnostics;
+using Nevermore.IntegrationTests.SetUp;
+using NUnit.Framework;
+
+namespace Nevermore.IntegrationTests.RelationalTransaction;
+
+public class TransactionTimerFixture : FixtureWithRelationalStore
+{
+    class MockTransactionLogger : ITransactionLogger
+    {
+        public List<(long duration, string transactionName)> Entries { get; } = new();
+        
+        public void Write(long duration, string transactionName)
+        {
+            Entries.Add((duration, transactionName));
+        }
+    }
```

Comment:
No `Moq` or `Substitute` available so made a simple mock logger to verify the method calls.